r-build.md

```diff
@@ -0,0 +1,177 @@
+---
+highlighter: none
+layout: default
+title: Building R in HTC
+---
+
+# Overview
+
+**This guide will provide instructions for compiling base R from source code and is intended 
+for compiling versions of R that are currently not provided via the CHTC Squid Web Proxy**
+
+Before compiling R, please take a moment to review our [R Jobs](http://chtc.cs.wisc.edu/r-jobs.shtml) guide to see what versions of R are currently available on the HTC.
```

Comment:
"on the HTC" seems like odd phrasing... maybe "currently available via CHTC" to mirror the bolded part above?